### PR TITLE
Add descriptions for homi packages

### PR DESCRIPTION
**Pull request by andybclee**
## Proposed changes

- This PR adds doc.go files for all packages under cmd/homi directory.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/homi/docker/doc.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+
+/*
+docker package contains compose and service packages which are to be used to create a docker-compose.yml with given options
```
Comment (KimKyungup) [inline cmd/homi/docker/doc.go]: 'contains compose' is strange.
```suggestion
docker package contains service packages which are to be used to create a docker-compose.yml with given options
```

File: cmd/homi/docker/compose/doc.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+
+/*
+compose package provides homi object which is to be used to create a docker-compose.yml
+
+Source Files
+
+Each file contains following contents
+ - homi.go : Provides `Homi` and create an assembled docker-compose.yml
```
Comment (jeonghonah) [inline cmd/homi/docker/compose/doc.go]:
```suggestion
 - homi.go : Provides `Homi` and creates an assembled docker-compose.yml
```

File: cmd/homi/docker/doc.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+
+/*
+docker package contains service packages which are to be used to create a docker-compose.yml with given options
+docker support is enabled by default when using `setup` command and can be set explicitly by giving "docker" as a type argument
```
Comment (jeonghonah) [inline cmd/homi/docker/doc.go]:
```suggestion
docker support is enabled by default when using `setup` command and can be set explicitly by giving `docker` as a type argument
```

File: cmd/homi/genesis/doc.go [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+//
+
+/*
+genesis package provides utility functions to generate new genesis file to init a chain
+
+Source Files
+
+Each file contains following contents
+ - genesis.go : Provide functions to make a new genesis object
+ - options.go : Provide utility functions to generate each part in a genesis file such as a list of validators
```
Comment (jeonghonah) [inline cmd/homi/genesis/doc.go]:
```suggestion
 - options.go : Provides utility functions to generate each part in a genesis file such as a list of validators
```

**Pull request merged**